runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java

```diff
@@ -44,6 +45,7 @@ default boolean hideSkillIcons()
 		return false;
 	}
 
+	@Alpha
```

Comment:
I don't think this works because it looks like the Widgets don't support alpha text colors? Does it work for you?